src/lando/main/models/profile.py

```diff
@@ -44,6 +46,32 @@ class Meta:
     # User info fetched from SSO.
     userinfo = models.JSONField(default=dict, blank=True)
 
+    # Encrypted Phabricator API token.
+    encrypted_phabricator_api_key = models.BinaryField(default=b"", blank=True)
+
+    @property
+    def _fernet(self):
+        """A class providing encryption and decryption based on the key in settings."""
+        return Fernet(settings.ENCRYPTION_KEY)
```

Comment:
Nit: we are generating a new `Fernet()` object each time we access `self._fernet`. Since this only relies on the top-level `settings.ENCRYPTION_KEY`, we could move it outside of the `Profile` class, along with the encrypt/decrypt functions which rely on it (either in the top-level of this module, their own module, their own class etc etc).

Reply:
I would prefer to avoid initializing variables that depend on a settings value on a module (top) level, that can cause other problems and I generally dislike that approach.

I think this still belongs in the Profile class, but I can move it to be outside of a property so that it gets initialized only once per instance, rather than on every call.